lib/rubocop/cop/mixin/allowed.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Common functionality for checking whitelisted methods & class names.
+    module Allowed
```

Comment:
I think that the methods in this module may be inlined into `DynamicFindBy` class.